src/components/component-playground.jsx

```diff
@@ -270,6 +270,10 @@ module.exports = React.createClass({
     window.addEventListener('resize', this.onWindowResize);
 
     this._updateContentFrameOrientation();
+
+    if(this.props.component) {
+      this.refs["componentName-" + this.props.component].getDOMNode().scrollIntoView(); 
```

Comment:
Since this is just for us and we don't really care for browser support (we all use Chrome), can you add `behavior: "smooth"` as option to enable animations?